tests/wamr-test-suites/test_wamr.sh

```diff
@@ -469,6 +472,43 @@ function wasi_test()
     echo "Finish wasi tests"
 }
 
+function wasi_certification_test()
+{
+    echo  "Now start wasi tests"
+    
+    cd ${WORK_DIR}
+    if [ ! -d "wasi-testsuite" ]; then
+        echo "wasi not exist, clone it from github"
+        git clone -b prod/testsuite-base \
+            --single-branch https://github.com/WebAssembly/wasi-testsuite.git
+    else
+        echo "wasi exists, pulling updates from github"
+        pushd wasi-testsuite
+        git fetch
+        git reset --hard origin/prod/testsuite-base
```

Comment:
I'd suggest at least for now we point to a specific commit. Using always the latest testsuite is a bit risky and might block the pipeline, especially that there's a lot going on in the wasi-testsuite repo now. We might want to choose to use always the latest version in the future, but I'd point to specific commit for now.